daemon/stats.go

```diff
@@ -126,6 +129,20 @@ func (daemon *Daemon) ContainerStats(ctx context.Context, prefixOrName string, c
 			if !config.Stream {
 				return nil
 			}
+		case <-time.After(statsInterval):
```

Comment:
According to explaination: https://golang.org/pkg/time/#Ticker , I don't think `Ticker is what I want, It will send the time periodically, what I want is a `timeout` mechanism.